api/system.cc

```diff
@@ -182,6 +183,17 @@ void set_system(http_context& ctx, routes& r) {
         apilog.info("Profile dumped to {}", profile_dest);
         return make_ready_future<json::json_return_type>(json::json_return_type(json::json_void()));
     }) ;
+
+    hs::get_highest_supported_sstable_version.set(r, [&ctx](const_req req) -> sstring {
+        auto supported_features = ctx.db.local().features().supported_feature_set();
+        for (auto format : {"ME_SSTABLE_FORMAT", "MD_SSTABLE_FORMAT", "MC_SSTABLE_FORMAT"}) {
+            if (auto it = supported_features.find(format); it != supported_features.end()) {
+                return sstring(format, 2);
```

Comment:
nit: lower case would be nicer

Reply:
If you use `sstables_manager::get_highest_supported_format()` then the returned value can be simply formatted into a string. See sstables/version.hh:
```c++
template <>
struct fmt::formatter<sstables::sstable_version_types> : fmt::formatter<string_view> {
    template <typename FormatContext>
    auto format(const sstables::sstable_version_types& version, FormatContext& ctx) const {
        return fmt::format_to(ctx.out(), "{}", sstables::version_string.at(version));
    }
};
```

and:
```c++
const std::unordered_map<sstable_version_types, sstring, enum_hash<sstable_version_types>> version_string = {
    { sstable_version_types::ka , "ka" },
    { sstable_version_types::la , "la" },
    { sstable_version_types::mc , "mc" },
    { sstable_version_types::md , "md" },
    { sstable_version_types::me , "me" },
};
```

(and then it would be lower case :))

Reply:
BTW, could use `seastar::to_sstring()` for this purpose.